openedx/adg/lms/our_team/models.py

```diff
@@ -9,6 +9,24 @@
 from .helpers import validate_profile_image_size
 
 
+class TeamMembersManager(models.Manager):
+    """
+    Manager for all the `Team Member` type of members
+    """
+
+    def get_queryset(self):
+        return super().get_queryset().filter(member_type=OurTeamMember.TEAM_MEMBER)
+
+
+class BoardOfTrusteesManager(models.Manager):
+    """
+    Manager for all the `Board of Trustees` type of members
+    """
+
+    def get_queryset(self):
+        return super().get_queryset().filter(member_type=OurTeamMember.BOARD_MEMBER)
```

Comment:
Instead of writing separate managers, how about creating just one manager and adding these two as the methods in there and then assigning that manager to default objects